CHANGELOG.asciidoc

```diff
@@ -3,6 +3,101 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.4.1]
+=== Beats version 7.4.1
+https://github.com/elastic/beats/compare/v7.4.0...v7.4.1[View commits]
+
+==== Breaking changes
```

Comment:
I presume we will clean this empty section before the release?

Reply:
Yes, I'll remove empty sections before the release.